src/details/ArborX_DetailsTreeTraversal.hpp

```diff
@@ -62,10 +62,14 @@ struct TreeTraversal<BVH, Predicates, Callback, SpatialPredicateTag>
     }
     else
     {
-      Kokkos::parallel_for("ArborX::TreeTraversal::spatial",
-                           Kokkos::RangePolicy<ExecutionSpace>(
-                               space, 0, Access::size(predicates)),
-                           *this);
+      Kokkos::RangePolicy<ExecutionSpace> policy(space, 0,
+                                                 Access::size(predicates));
+      // FIXME_SYCL
+#ifdef KOKKOS_ENABLE_SYCL
+      if constexpr (std::is_same_v<ExecutionSpace, Kokkos::Experimental::SYCL>)
+        policy.set_chunk_size(1024);
```

Comment:
Can you please add a comment on the considerations that went into choosing 1024?

Reply:
Not quite sure if there is much behind it rather than that choosing the maximum gave much better performance (than 32 which is what the compiler chose) and I couldn't find a different workgroup size that gave better results (but results don't differ much choosing between 256 and 1024 threads per workgroup).

Reply:
Can you please post the results in this issue for different workgroup sizes?

Reply:
I'll share it privately.

Reply:
Please make sure to cc me